cyder/core/system/models.py

```diff
@@ -41,7 +41,6 @@ def details(self):
         data = super(System, self).details()
         data['data'] = [
             ('Name', 'name', self),
-            ('Ctnr', 'ctnr', self.ctnr),
```

Comment:
Doing this removes a feature I rather liked actually, which is that you could see the System's container in the list view. This is only useful in the global container and the search view, of course, but I think it was useful enough to justify its existence.